confgenerator/confgenerator_test.go

```diff
@@ -89,6 +90,14 @@ func TestGenerateConfsWithValidInput(t *testing.T) {
 }
 
 func testGenerateConfsWithValidInput(t *testing.T, platform platformConfig) {
+	getExecutableFolder = func() (string, error) {
+		if runtime.GOOS != "windows" {
```

Comment:
We generate both windows and linux configs on both platforms, so you can't look at `runtime.GOOS` here :(

Reply:
Looks like this has been addressed in https://github.com/GoogleCloudPlatform/ops-agent/commit/91068764f0d1fb3c7b70bf7dfc3a2134c1f7e719#diff-85d8cd0bc6302678b49b04ef15857774aaab001501d09e2df6424d9e20f6ad3d.